src/main/java/frc/robot/controllers/ControllerBase.java

```diff
@@ -20,9 +20,20 @@ public double getMoveRequest() {
     return moveRequest;
   }
 
+  public double getShootPowerCells() {
+    return Utils.deadzone(xbox.getTriggerAxis(Hand.kRight), 0.2);
+  }
+
   public double getTurnRequest() {
     double turnRequest = xbox.getX(XboxController.Hand.kRight);
     return turnRequest;
   }
 
+  public boolean getIncHoodPos() {
+    return xbox.getPOV() == 180;
+  }
+
+  public boolean getDecHoodPos() {
+    return xbox.getPOV() == 0;
+  }
```

Comment:
It feels confusing that half of the inputs are defined in the RobotContainer and half of the inputs are defined in the DriveController.
This is why in previous years we didn't use JoystickButtons, instead using [BooleanTriggers](https://github.com/momentumfrc/2019DeepSpace/blob/1c7e3cb6882befb5f5d0f91fd71fd124007be35b/2019RobotCodeDeepSpace/src/main/java/frc/robot/OI.java#L54-L60) linked to the DriveController class.